### PR TITLE
Add celoDUCO (experimental)

### PR DESCRIPTION
**Pull request by ygboucherk**
Duino-Coin (DUCO) is an arduino mineable coin, and celoDUCO is its celo peg (after bscDUCO and maticDUCO).
As liquidity is still veeery low, token is added as experimental !

### CODE REVIEW

File: ubeswap.token-list.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "Ubeswap",
+  "name": "Unknown List",
```
Comment (BriungRi) [inline ubeswap.token-list.json:2]: Why was this changed?

Comment (ygboucherk) [reply ubeswap.token-list.json:2]: idk

**Comment by ygboucherk** [comment]
Anyway replaced names manually

File: ubeswap.token-list.json (hunk 1 above)
Comment (ygboucherk) [reply ubeswap.token-list.json]: Anyway fixed manually for now
